generators/app/index.js

```diff
@@ -39,11 +39,49 @@ module.exports = fountain.Base.extend({
     },
 
     conf: function () {
-      this.fs.copyTpl(
-        this.templatePath('conf'),
-        this.destinationPath('conf'),
-        { framework: this.props.framework }
-      );
+      const lit = this.lit;
```

Comment:
NOTE: lit is define here https://github.com/FountainJS/fountain-generator/blob/proto-json-conf/lib/file-utils.js#L39